lib/fluent/plugin/output.rb

```diff
@@ -1115,6 +1149,21 @@ def try_flush
         end
       end
 
+      def backup_chunk(chunk, using_secondary, delayed_commit)
+        unique_id = dump_unique_id_hex(chunk.unique_id)
+        safe_plugin_id = plugin_id.gsub(/[ \/\\:]/, '_')
```

Comment:
This feature assumes to work on Windows?
If so, we should add more prohibiting to use characters:

`/ > < ? : " \ * | ;` are not permitted to use on NTFS.